web/src/app/administration/components/access-items-management/access-items-management.component.html

```diff
@@ -1,177 +1,136 @@
 <div class="access-items-management panel panel-default">
-
-  <div class="panel-heading">
-    <h4 class="panel-header">Access items management</h4>
-  </div>
-
   <div class="panel-body">
-    <div class="col-md-6 col-md-offset-3 margin">
+    <div class="panel-body__typeahead">
       <taskana-shared-type-ahead name="accessIdSelected" [(ngModel)]="accessIdSelected"
         placeHolderMessage="Search for access id..." (selectedItem)="onSelectAccessId($event)" displayError=true
         isRequired="false">
       </taskana-shared-type-ahead>
     </div>
-
-    <!-- NO ID PLACEHOLDER -->
-    <div *ngIf="!accessItemsForm" class="center-block no-detail col-xs-12">
+    <div *ngIf="!accessItemsForm" class="center-block no-detail">
       <h3 class="grey">Select an access id</h3>
       <svg-icon class="empty-icon" src="./assets/icons/users.svg"></svg-icon>
     </div>
-
-    <!-- CONTENT -->
-    <div *ngIf="accessItemsForm" class="row col-xs-12">
-      <ng-form [formGroup]="accessItemsForm">
-        <!-- TABLE WITH ACCESS ID -->
-        <table id="table-access-items" class="table table-striped table-center">
-          <thead>
-            <!-- TABLE FIRST ROW -->
-            <tr>
-              <th>
-                <taskana-shared-sort [sortingFields]="sortingFields" (performSorting)="sorting($event)"
-                  menuPosition="left">
-                </taskana-shared-sort>
-              </th>
-              <th class="text-align min-width">Workbasket Key</th>
-              <th colspan="2" class="text-align">Access Id</th>
-              <th>Read</th>
-              <th>Open</th>
-              <th>Append</th>
-              <th>Transfer</th>
-              <th>Distribute</th>
-              <ng-container *ngFor="let customField of customFields$ | async">
-                <th *ngIf="customField.visible">
-                  {{customField.field}}
+    <mat-expansion-panel *ngIf="accessItemsForm" class="expansion-panel" (opened)="panelOpenState = true"
```

Comment:
Àccording to BEM naming convention, the name 'expansion-panel' should be 'panel-body__expansion-panel'.